apps/ui/src/components/pages/manage/DashboardFilter.tsx

```diff
@@ -58,12 +60,12 @@ const DashboardFilter = ({ onFilterChange, availableStates }: DashboardFilterPro
 			return {
 				name: translate(`application.states.${possibleState}`),
 				amount: availableStates.find((state) => state.key === possibleState)?.amount || 0,
-				key: possibleState as FilterKeyType,
+				key: possibleState as FilterKeys,
```

Comment:
This block can be changed to avoid use of `as`: 

```
		...Object.values(ApplicationStates).map((possibleState: FilterKeys): Filter => {
			return {
				name: translate(`application.states.${possibleState}`),
				amount: availableStates.find((state) => state.key === possibleState)?.amount || 0,
				key: possibleState,
			};
		}),
```